### ISSUE TITLE
Layoutlmft is not convergence if the pytorch is higher or equal to 1.8

### ISSUE THREAD

**Issue by 14H034160212** [microsoft/unilm#894]
**Describe the bug**
Model I am using (LayoutLMFT.):

The problem arises when using:
* [https://github.com/microsoft/unilm/blob/master/layoutlmft/requirements.txt ] the official example scripts
* my own modified scripts: (give details below)
```
-f https://download.pytorch.org/whl/torch_stable.html
-f https://dl.fbaipublicfiles.com/detectron2/wheels/cu101/torch1.8/index.html

datasets==1.6.2
torch==1.8.0+cu101
torchvision==0.9.0+cu101
transformers==4.5.1
detectron2==0.4
seqeval==1.2.2
wandb
```

When I upgrade the pytorch version and the model is not convergence during training. The green line is the loss curve for the pytorch==1.8, blue curve is the one which does not use GPU for training and the red curve is the one which use GPU for training with the pytorch given by the paper.
![image](https://user-images.githubusercontent.com/23516191/196823595-a201ebcd-6d39-45b9-8b0c-b9337f395a75.png)

**To Reproduce**
1. Upgrade the pytorch version following my above script.
2. Run the command `CUDA_VISIBLE_DEVICES=0 python -m examples.run_xfun_re --model_name_or_path microsoft/layoutxlm-base --output_dir /root/unilm/layoutlmft/updated_torch_XLM_RE_2 --do_train  --do_eval --lang zh   --max_steps 5000   --per_device_train_batch_size 1 --warmup_ratio 0.1 --report_to wandb --per_device_eval_batch_size 1`

**Expected behavior**
The expected result is the training can convergence, but the actual result is the model is not convergence.
- Platform: Linux
- Python version: 3.7
- PyTorch version (GPU?): greater or equal to 1.8

**Comment by 14H034160212** [comment]
The issue has been fixed when I try to use training batch size=2.
